src/main/java/blackjack/domain/BlackJackGame.java

```diff
@@ -0,0 +1,115 @@
+package blackjack.domain;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.ShufflingMachine;
+import blackjack.domain.card.Deck;
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Participant;
+import blackjack.domain.participant.Player;
+import blackjack.domain.participant.Players;
+import blackjack.domain.participant.Result;
+
+public class BlackJackGame {
+
+    private static final int NUMBER_OF_INITIAL_CARD = 2;
+
+    private final Dealer dealer;
+    private final Players players;
+
+    public BlackJackGame(final String inputNames) {
+        this.dealer = new Dealer();
+        this.players = new Players(inputNames);
+    }
```

Comment:
Dealer에는 속성이 없는걸까요?! 
단순히 인스턴스 생성만 해준다면, 외부에서 받아와도 될 것 같아서요!! 

```suggestion
    public BlackJackGame(Dealer dealer, final String inputNames) {
        this.dealer = dealer;
        this.players = new Players(inputNames);
    }
```

Reply:
생성자 주입을 사용하는 것이 더 좋아보이네요!
수정하였습니다.